roles/scp_http_int/tasks/http_int_credential.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Get the public key from Jenkins server
-  shell: cat /root/ci-logs/ci-logs.pub
+  shell: "{{cat /root/{{ target_name }}/{{ target_name }}.pub}}"
```

Comment:
No need to add '{{' at the start of the string. I didn't notice before pushing, sorry (but it broke nothing, so not a big deal). I am fixing it.